.github/workflows/docker-publish-njsPC-linux.yml

```diff
@@ -11,71 +11,22 @@ jobs:
   build-and-push:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-    
-      - name: Checkout code from tagyoureit/nodejs-poolcontroller
-        uses: actions/checkout@v3
-        with:
-          repository: tagyoureit/nodejs-poolcontroller
-          ref: master  # or specify the branch or tag to pull from
-          path: nodejs-poolcontroller
-
-      - name: Checkout code from rstrouse/nodejs-poolcontroller-dashpanel
-        uses: actions/checkout@v3
-        with:
-          repository: rstrouse/nodejs-poolcontroller-dashpanel  # 
-          ref: master  # Specify the branch or tag to pull from
-          path: nodejs-poolcontroller-dashpanel  # Specify the directory to checkout the code into
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        
       - name: Login to Docker Hub
         uses: docker/login-action@v3
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
       - name: Build and push combined Docker image
-        uses: docker/build-push-action@v4
+        uses: docker/build-push-action@v6
         with:
-          context: .
-          file: ./.docker/Dockerfile.linux
           push: true
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
           tags: |
             tagyoureit/njspc-dp-combined-app:latest
```

Comment:
Please rename to remove the combined tag.

Reply:
I've renamed to just njspc, please let me know if you'd like a different name. You'll need to setup the repository on dockerhub before we merge this.